content/index.njk

```diff
@@ -19,7 +19,7 @@ layout: layouts/base.njk
 	<div id="social_media_links">
 		<p>
 			Join us on
-			<a href="https://stuyactivities.org/linux"> StuyActivities </a>
+			<a href="https://stuyactivities.org/linux"> Stuy Activities </a>
```

Comment:
StuyActivities is usually shown with one word

Reply:
Should be good now :+1: